variables.tf

```diff
@@ -74,7 +74,7 @@ variable "tfc_agent_log_level" {
 
 variable "tfc_agent_data_dir" {
   type        = string
-  default     = "~/.tfc-agent"
+  default     = null
```

Comment:
Do you need to change the existing behavior here? Or can you keep the default as is and then pass in `null` yourself? I'd rather see the latter so that we don't have a breaking change for some folks using this module, but if you have a good reason to utilize this then feel free to disagree.

Reply:
1. the current default value causes errors so the agent doe not run jobs with this setting
2. I dont know why you would set a custom data dir location, but i left it in so if you realy want you still can set a custom location
3. setting this to default to null causes the agent uses the default location which always runs without errors